functions/global-setup.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
+const { setup: setupDevServer } = require('jest-dev-server')
+
+module.exports = async function globalSetup() {
+  globalThis.servers = await setupDevServer({
+    command: `yarn start:wrangler`,
+  })
+  await new Promise((r) => setTimeout(r, 30000))
```

Comment:
this is just waiting 30s before starting tests? could we do a `wait-port` type thing instead?

Reply:
I originally tried doing a `port-ready`, but for some reason it didn't seem to work with `wrangler` so I just defaulted to a wait which worked.

Reply:
Will look into potentially other libraries

Reply:
`wait-port` worked so jut pushed that into the newest commit!